### PR TITLE
FindWrapper children and childAt functions

### LINKED ISSUE #60: children and childAt functions for FindWrapper
Implement `children` and `childAt` functions for `FindWrapper`.

### PR DESCRIPTION
**Pull request by Deseteral**
Resolves #60

### CODE REVIEW

File: src/preact-render-spy.js [hunk 1]
```diff
@@ -240,6 +246,31 @@ class FindWrapper {
       .length > 0;
   }
 
+  childAt(index) {
+    verifyFoundNodes(this);
```
Comment (gnarf) [inline src/preact-render-spy.js]: couldn't this whole method be implemented as `return this.children().at(index)` ?

File: README.md [hunk 2]
```diff
@@ -161,6 +161,12 @@ expect(context.component()).toBeInstanceOf(MyComponent);
 ### `FindWrapper#contains(vdom)`
 Searches for any children matching the vdom or text passed.
 
+### `FindWrapper#childAt(index)`
+Returns `FindWrapper` with child at given index.
```
Comment (gnarf) [inline README.md]: Might be worth saying it's effectively `.children().at(index)`

File: README.md [hunk 3]
```diff
@@ -161,6 +161,12 @@ expect(context.component()).toBeInstanceOf(MyComponent);
...
+
+### `FindWrapper#children()`
+Returns `FindWrapper` with children of current wrapper.
```
Comment (gnarf) [inline README.md]: Perhaps a small usage example if you can think of one?

File: src/preact-render-spy.js (hunk 1 above)
Comment (Deseteral) [reply src/preact-render-spy.js]: Done

File: README.md (hunk 2 above)
Comment (Deseteral) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Deseteral) [reply README.md]: Done

**Comment by Deseteral** [comment]
Please take a look now

**Pull request merged**